libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/PredictiveCacheControllerTest.kt

```diff
@@ -1,5 +1,5 @@
 package com.mapbox.navigation.ui.maps
-
+/*
```

Comment:
This should be removed. Already addressed in https://github.com/mapbox/mapbox-navigation-android/pull/4268

Reply:
Ah, a merge issue. Good catch :+1: